[2025-09-28T00:06:46.533+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-28T00:06:46.538+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hw1.collect_metrics_model scheduled__2025-09-27T00:00:00+00:00 [queued]>
[2025-09-28T00:06:46.541+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hw1.collect_metrics_model scheduled__2025-09-27T00:00:00+00:00 [queued]>
[2025-09-28T00:06:46.541+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-28T00:06:46.546+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): collect_metrics_model> on 2025-09-27 00:00:00+00:00
[2025-09-28T00:06:46.548+0000] {standard_task_runner.py:64} INFO - Started process 89074 to run task
[2025-09-28T00:06:46.550+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hw1', 'collect_metrics_model', 'scheduled__2025-09-27T00:00:00+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/kudryakov_hw1.py', '--cfg-path', '/tmp/tmpw2is8dud']
[2025-09-28T00:06:46.551+0000] {standard_task_runner.py:91} INFO - Job 240: Subtask collect_metrics_model
[2025-09-28T00:06:46.567+0000] {task_command.py:426} INFO - Running <TaskInstance: hw1.collect_metrics_model scheduled__2025-09-27T00:00:00+00:00 [running]> on host 2b301b74205f
[2025-09-28T00:06:46.597+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='VLADISLAV KUDRYAKOV' AIRFLOW_CTX_DAG_ID='hw1' AIRFLOW_CTX_TASK_ID='collect_metrics_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-27T00:00:00+00:00'
[2025-09-28T00:06:46.598+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-28T00:06:46.602+0000] {logging_mixin.py:188} INFO - collect_metrics_model
[2025-09-28T00:06:46.602+0000] {kudryakov_hw1.py:209} INFO - Начали расчет метрик
[2025-09-28T00:06:46.609+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2025-09-28T00:06:46.609+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2025-09-28T00:06:53.885+0000] {logging_mixin.py:188} INFO - {'r_squared': 0.8503834409713716, 'RMSE': 0.19244684232083303} metrics
[2025-09-28T00:06:54.172+0000] {kudryakov_hw1.py:228} INFO - Успешно Завершен расчет метрик
[2025-09-28T00:06:54.190+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-28T00:06:54.190+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-28T00:06:54.197+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hw1, task_id=collect_metrics_model, run_id=scheduled__2025-09-27T00:00:00+00:00, execution_date=20250927T000000, start_date=20250928T000646, end_date=20250928T000654
[2025-09-28T00:06:54.242+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-28T00:06:54.254+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-28T00:06:54.255+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
